classifai.php

```diff
@@ -186,3 +191,17 @@ function( $plugin_info, $http_response = null ) {
 		// @codingStandardsIgnoreEnd
 	}
 }
+
+/**
+ * Show dev version notice on ClassifAI pages if necessary.
+ */
+function classifai_dev_notice() {
+	if ( 0 === strpos( get_current_screen()->parent_base, 'classifai' ) ) {
+		?>
+		<div class="notice notice-warning">
+		<?php /* translators: %1$s: CLI install commands, %2$s: classifai url */ ?>
+		<p><?php echo wp_kses_post( sprintf( __( 'You appear to be running a development version of ClassifAI. Certain features may not work correctly without regularly running %1$s. If you&rsquo;re not sure what this means, you may want to <a href="%2$s">download and install</a> the stable version of ClassifAI instead.', 'distributor' ), '<code>composer install && npm install && npm run build</code>', 'https://classifaiplugin.com/' ) ); ?></p>
```

Comment:
The encoded apostrophe caught my eye here. At first I was going to say use `If you´re not sure` here instead of `If you&rsquo;re not sure`... instead thinking maybe `If you are not sure`?

Reply:
I copied this out of Distributor :) I guess I'm a dinosaur who still believes in encoding special characters?

Reply:
And the reason I prefer it that way and I think I will keep it that way is because it comes across as friendlier IMO, which important after an intimidating-sounding sentence.

Reply:
👍 nice!